File: .sdk/tm/ts/test/exists.test.ts

import { test, describe } from 'node:test'
import { equal } from 'node:assert'


import { NobelPrizeSDK } from '..'


describe('exists', async () => {

  test('test-mode', async () => {
    const testsdk = await NobelPrizeSDK.test()
    equal(null !== testsdk, true)
  })

})

File: rb/test/exists_test.rb
# NobelPrize SDK exists test

require "minitest/autorun"
require_relative "../NobelPrize_sdk"

class ExistsTest < Minitest::Test
  def test_create_test_sdk
    testsdk = NobelPrizeSDK.test(nil, nil)
    assert !testsdk.nil?
  end
end
